rotkehlchen/chain/ethereum/modules/constants.py

```diff
@@ -9,4 +10,11 @@
     CPT_UNISWAP_V1,
     CPT_UNISWAP_V2,
     CPT_UNISWAP_V3,
+    CPT_SUSHISWAP_V2,
```

Comment:
hey so if it's not sushiswap v2 and only v1 can we get rid of the V2 tag?

Reply:
We saw that there is a v2 of the masterchef contract so we will keep it as it is